code/game/turfs/flooring/flooring_decals.dm

```diff
@@ -21,6 +21,8 @@
 		if(!T.decals) T.decals = list()
 		T.decals |= floor_decals[cache_key]
 		T.add_overlay(floor_decals[cache_key])
+		if(desc)
+			T.desc += "<br>There is a [src] on \the [T] with the following inscription:<br><i>[desc]</i>"
```

Comment:
This should be `\a [src]`. Otherwise you can end up with weird abominations in text.

Also, I think having the source reference it in its description (the `[T]`) is a bit weird. Replacing it with `it` might work out better.

`<br>There is \a [src] on it with the following inscription:<br><i>[desc]</i>`